### ISSUE TITLE
compatibility with SexyMap

### ISSUE THREAD

**Issue by Trajin** [brittyazel/Neuron#66]
The author of SexyMap addon suggested that I ask you to use LibDBIcon for the minimap button and that should make it detectable by SexyMap.

**Comment by brittyazel** [comment]
I'll look into that. This is the first I've heard of LibDBIcon, but I'm happy to do it. I have a couple of big bugs I need to address first though

**Comment by cysteve** [comment]
currently with sexymap, if you left click the button to configure,  then left click again to exit configuring, then the button gets very low strata or something,  it goes behind the minimap frame.  think reload fixes it until you click the button again.

**Comment by Trajin** [comment]
What I was hoping for is that it shows up in SexyMap's addon button list so I can set it so it will show on hover (mouseover).   It will be hidden the rest of the time.

**Comment by brittyazel** [comment]
I haven't looked into this yet, I was trying to get some major bugs fixed. I'll see if I can replace some of our old minimap code with LibDBIcon stuff. I still need to figure all that out

**Comment by Trajin** [comment]
I was just trying to clarify what i was hoping for.    No rush on this.   Keep up the good work.  Looks like the vehicle/last button bug is fixed.

Sent from my iPhone

On Apr 6, 2018, at 10:10 PM, Britt Yazel <notifications@github.com<mailto:notifications@github.com>> wrote:


I haven't looked into this yet, I was trying to get some major bugs fixed. I'll see if I can replace some of our old minimap code with LibDBIcon stuff. I still need to figure all that out

**Comment by brittyazel** [comment]
Fixed, we are now making use of LibDBIcon. Thanks for pointing me in that direction. Let me know how this works for you

**Comment by brittyazel** [comment]
Awesome. I changed the commands a bit slightly to activate all the different modes on the minimap button, so now instead of alt+click or shift+right click or, middle click, it is just 1) Left Click 2) Right Click 3) Shift+Left Click 4) Shift + Right click. 
Does the way I have it make sense to you? I tried to keep it simple and to the point, and so far it seems to be working the way I intended lol

**Comment by Trajin** [comment]
I noticed the change when i was working on some alts last night.   The icon name within sexymap is showing as orbbutton instead of being called neuron.   Not a big thing, just letting you know.   It auto hides like I wanted.

Sent from my iPhone

On Apr 8, 2018, at 6:02 PM, Britt Yazel <notifications@github.com<mailto:notifications@github.com>> wrote:


Awesome. I changed the commands a bit slightly to activate all the different modes on the minimap button, so now instead of alt+click or shift+right click or, middle click, it is just 1) Left Click 2) Right Click 3) Shift+Left Click 4) Shift + Right click.
Does the way I have it make sense to you? I tried to keep it simple and to the point, and so far it seems to be working the way I intended lol
